spring-app/src/main/java/pl/edu/agh/cqm/controller/MonitoringController.java

```diff
@@ -70,4 +70,19 @@ public AllParametersResponseDTO getAll(
                 monitoringService.getThroughputSamples(startDate, endDate)
         );
     }
+
+    @PutMapping("/update-parameters")
+    public void ParametersDTO(
+            @Valid ConfigParametersDTO configParametersDTO
+    ) {
+        List<String> cdns = configParametersDTO.getCdns();
+        updateParametersService.updateCdns(cdns);
+
+        int activeSamplingRate = configParametersDTO.getActiveSamplingRate();
+        int activeTestIntensity = configParametersDTO.getActiveTestIntensity();
+
+        int passiveSamplingRate = configParametersDTO.getPassiveSamplingRate();
+
+        updateParametersService.updateSampleParameters(activeSamplingRate, activeTestIntensity, passiveSamplingRate);
+    }
 }
```

Comment:
endpoint for getting current parameters is missing. It will be needed for displaying settings form in react app